lib/test/tasks/run_file_size_checks.rb

```diff
@@ -8,7 +8,7 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
   CONSTRAINTS = {
     'charts*.js' => (295..305),
     'check_ins*.css' => (10..15),
-    'check_ins*.js' => (191..201),
+    'check_ins*.js' => (197..207),
```

Comment:
I'm slightly surprised that we have to increase any of these file size allowances, but whatever.